runtime/codert_vm/cnathelp.cpp

```diff
@@ -3102,6 +3114,21 @@ fast_jitTypeCheckArrayStoreWithNullCheck(J9VMThread *currentThread, j9object_t d
 	return slowPath;
 }
 
+void* J9FASTCALL
```

Comment:
If this is the direct call version, you need to return the value from the function, not use the macro.